### ISSUE TITLE
(Bug report) Trilium fail to load

### ISSUE THREAD

**Issue by WPum** [zadam/trilium#3362]
### Trilium Version

0.56.2

### What operating system are you using?

Windows

### What is your setup?

Server access only

### Operating System Version

Windows 10 version 2004 with Firefox 107.0

### Description

Sometimes Trilium fail to load, showing a blank page with an error "ERROR when calling GET options: error - undefined".

Console log:

![image](https://user-images.githubusercontent.com/27683756/204700495-2e5f0e36-f735-4f37-befd-6b8c1ab2b387.png)

I can't find anything unusual in the backend log.

**Comment by zadam** [comment]
Hi, that kind of looks like some intermittent network error.

**Comment by WPum** [comment]
Hi, I deploy Trilium to localhost, so the network is unlikely to cause an error.

Luckily I found a way to reproduce this kind of error.

It seems that this error is related to rapid actions on the note interface. For example, I created two notes and switch between them rapidly. At the same time I refreshed the page by pressing F5, and then the error occurred.

https://user-images.githubusercontent.com/27683756/204969317-bc7555ba-a0b0-4506-8584-1831e19e2eeb.mp4
